src/lighthouse.php

```diff
@@ -453,4 +453,24 @@
          */
         'entities_resolver_namespace' => 'App\\GraphQL\\Entities',
     ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Tracing
+    |--------------------------------------------------------------------------
+    |
+    | Configuration for tracing support.
+    |
+    */
+
+    'tracing' => [
+        /*
+         * Driver used for tracing.
+         *
+         * Available options: 'apollo-tracing', 'federated-tracing'
+         *
+         * Will be removed in Lighthouse v7 and implicitly default to 'federated-tracing'.
+         */
+        'driver' => env('LIGHTHOUSE_TRACING', \Nuwave\Lighthouse\Tracing\ApolloTracing\ApolloTracing::NAME),
```

Comment:
```suggestion
        'driver' => env('LIGHTHOUSE_TRACING', Nuwave\Lighthouse\Tracing\ApolloTracing\ApolloTracing::NAME),
```

Reply:
I am not sure an `env` variable is needed, I don't see this changing per environment? We could just use the class names instead then, that would also allow the addition of custom classes because we would no longer need to hardcode driver name to classes.

Reply:
Changing the tracing driver per environment I don't think is too useful but it can be useful for example to enable/disable tracing if we add a `null` driver that does nothing. This can be useful for testing/local environment where tracing is only an overhead. If we remove use the class name directly, this can be implemented with a simple boolean flag.

Another thing to consider is keep the "driver" architecture to allow other drivers to be supported in the future (for example `opentelemetry`)

Reply:
The new federated tracing is enabled only when there is the `apollo-federation-include-trace` header, the old apollo tracing is always enabled.

Anyway conditionally registering the service provider is a good solution for enabling/disabling the tracing.